sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,237 @@
+  - Transaction fee must be estimated proportionally for the requested operation [2].
+- Collectively, the signers must coordinate to calculate and advertise the fee parameters of the system:
+  - A minimum sBTC peg-out.
+  - STX transaction fee for minting the sBTC. This fee is paid by the user and can be sponsored by a 3rd party.
+
+### sBTC Signer Eligibility Criteria
+
+Signers will run the sBTC binary in addition to the core Stacks signer software and must meet certain criteria in order to facilitate the reliable functioning of the sBTC protocol at all times.
+
+The following eligibility criteria will be used to identify the sBTC Signers:
+
+- Does the proposed sBTC signer have a demonstrable operating history which shows their experience and reliability in running blockchain services?
+- Has the proposed sBTC signer participated in running a Stacks signer instance on Stacks 2.5 testnet or mainnet, and can they provide metrics showing this (ex: amount of stacks stacked over past several cycles)?
+  * Note: The sBTC signer is a Stacks event observer, meaning that the experience of running a Stacks node signer directly translates to running an sBTC signer.
+- Does the proposed sBTC signer agree to use reasonable efforts to maintain >99% uptime on the sBTC Signer? 
+  * Note: This metric may be self-affirmed if independent verification is not possible, or confirmed by on-chain voting/stacking activity.*
+- Does the proposed sBTC signer commit to a direct communication channel to be set up with the sBTC core engineers in order to respond to urgent updates within 24 hours?
+- Has the signer made contributions to Bitcoin or the Stacks network over the past year that demonstrate their commitment to the growth and success of the network? Examples include, but are not limited to: publishing independent research, marketing, co-authoring a SIP, submitting a Stacks pull request/issue, providing feedback on Stacks core development, or contributing to a Stacks Working Group.
+
+The criteria described above will be used to identify sBTC Signers that are able to meet some or all of the responsibilities described in the previous section.
+
+### Selection Process
+The initial sBTC Signer Set will be finalized from the list of eligible Signers, based on the above criteria. The results will be published as a discussion in the [sBTC Working Group](https://github.com/stacks-network/sbtc/discussions/624).
+
+The selection process is as follows:
+1. **Nomination Phase**: Open a call for nominations within the community.
+2. **Evaluation & Community Feedback**: The proposed signer set will be published to provide transparency.
+3. **SIP Vote**: The community will vote on the signer criteria and proposed signer set.
+4. **Final Selection**: The final list of signers will be selected based on community feedback and successful completion of SIP-028.
+
+### Updating The sBTC Signer Set
```

Comment:
This directly contradicts the third paragraph of the `Specification` section.

Reply:
Updated the Specification section to correct this.